process/templates/PROJECT_NAME_sandbox_stage.md

```diff
@@ -0,0 +1,25 @@
+## Application for creating a new project at Sandbox stage
+
+### List of project maintainers
+The project must have a minimum of two maintainers with different organizational affiliations.
+  * "name, affiliation, GitHub ID"
+
+### Mission of the project
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+  * "description of the project mission"
+
+### IP policy and licensing due dilligence
+When contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF).
+  * "yes / no / not applicable. If yes, provide a link to the corresponding GitHub issue."
```

Comment:
Does this need to be a tri-state or can it be "Yes / not applicable"?

Reply:
dropping it to "Yes / not applicable" SGTM, but tagging @AevaOnline who I believe wrote this text

Reply:
The implied "no" would result in rejection of the application. 

I find that it is more inclusive of folks from other backgrounds (both experience levels and cultures) to make the implicit become explicit, so I favor the tri-state formulation of this even though it is semantically equivalent given a high context understanding.